### PR TITLE
#142 Updated validation and added unit tests

### PR DESCRIPTION
**Pull request by defectiveAi**
#142 Updated validation and added unit tests

### CODE REVIEW

**Review by FirestarJes — APPROVED** [review]

File: source/Energinet.DataHub.PostOffice.Application/Validation/DequeueCommandRuleSet.cs [hunk 1]
```diff
@@ -12,18 +12,22 @@
...
 // limitations under the License.
 
-using Energinet.DataHub.PostOffice.Application.Commands;
 using Energinet.DataHub.PostOffice.Application.Validation.Rules;
 using FluentValidation;
 
 namespace Energinet.DataHub.PostOffice.Application.Validation
 {
-    public class DataAvailableRuleSet : AbstractValidator<DataAvailableNotificationCommand>
+    public class DequeueCommandRuleSet : AbstractValidator<Commands.DequeueCommand>
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Application/Validation/DequeueCommandRuleSet.cs:20]: Add the using instead like in the two other `RuleSets`. For consistency.

File: source/Energinet.DataHub.PostOffice.Application/Validation/PeekCommandRuleSet.cs [hunk 2]
```diff
@@ -18,11 +18,11 @@
...
 namespace Energinet.DataHub.PostOffice.Application.Validation
 {
-    public class PeekRuleSet : AbstractValidator<PeekCommand>
+    public class PeekCommandRuleSet : AbstractValidator<PeekCommand>
     {
-        public PeekRuleSet()
+        public PeekCommandRuleSet()
         {
-            RuleFor(document => document.Recipient).SetValidator(new DocumentCannotHaveEmptyValue());
+            RuleFor(command => command.Recipient).NotEmpty().SetValidator(new GlobalLocationNumberValidationRule());
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.PostOffice.Application/Validation/PeekCommandRuleSet.cs:25]: Needs at least two line breaks ;)

Comment (defectiveAi) [reply source/Energinet.DataHub.PostOffice.Application/Validation/PeekCommandRuleSet.cs]: Done

**Review by michaelmajgaard — APPROVED** [review]

File: source/Energinet.DataHub.PostOffice.Application/Validation/DequeueCommandRuleSet.cs (hunk 1 above)
Comment (defectiveAi) [reply source/Energinet.DataHub.PostOffice.Application/Validation/DequeueCommandRuleSet.cs:20]: Not possible, we have two DequeueCommand, must qualify.

**Pull request merged**